ui/components/app/nfts-items/nfts-items.js

```diff
@@ -181,34 +191,79 @@ export default function NftsItems({
                   key={`nft-${i}`}
                   className="nfts-items__item-wrapper"
                 >
-                  <Card
-                    padding={0}
-                    justifyContent={JustifyContent.center}
-                    className="nfts-items__item-wrapper__card"
-                  >
-                    {nftImage ? (
-                      <button
-                        className="nfts-items__item"
-                        style={{
-                          backgroundColor,
-                        }}
-                        onClick={handleImageClick}
-                      >
-                        <img
-                          className="nfts-items__item-image"
-                          data-testid="nft-image"
-                          src={nftImage}
-                          alt={nftImageAlt}
+                  {process.env.MULTICHAIN ? (
+                    <BadgeWrapper
+                      badge={
+                        <AvatarNetwork
+                          size={Size.SM}
+                          name={currentChain?.nickname}
```

Comment:
There should always be a `currentChain` so I don't think that the optional chaining is necessary.

Reply:
Fixed, thanks!